### ISSUE TITLE
Allow JsonMap to serialize maps stored within volumes

### ISSUE THREAD

**Issue by Brett208** [OutpostUniverse/JsonMap#12]
Might be nice to not have to unpack maps to serialize them?

**Comment by DanRStevens** [comment]
Agreed. Not too sure on how the command line parameters will work for that. Maybe separate out the folder form the filename? Or just assume it may be part of a VOL file within the directory given.
